### ISSUE TITLE
Debian install of 3.0.2 fails: "Database doesn't match reference schema"

### ISSUE THREAD

**Issue by dgilmour77** [bluecherrydvr/bluecherry-apps#387]
Newly created Debian Buster instance, installed mariadb server with empty root password, checkout v3.0.2, run scripts/build_pkg_native.sh.  Fails with loaded database scheme mismatch

**Comment by antonsviridenko** [comment]
Bluecherry is not compatible with MariaDB, you have to install MySQL server from Oracle instead

**Comment by curtishall** [comment]
@dgilmour77 You can try this for buster to install mysql-server and the Bluecherry (v3 - beta)

```
    apt-get -y update
    apt-get -y install gnupg
    wget -q https://dl.bluecherrydvr.com/key/bluecherry.asc -O- | apt-key add -
    wget -q http://repo.mysql.com/RPM-GPG-KEY-mysql -O- | apt-key add -
    wget --no-check-certificate --output-document=/etc/apt/sources.list.d/bluecherry-stretch.list https://dl.bluecherrydvr.com/sources.list.d/bluecherry-buster-unstable.list
    apt-get -y update
    apt-get -y install bluecherry
```

**Comment by dgilmour77** [comment]
Curtis, Anton, thanks to both of you for the responses and the suggestion of where to source mysql.  I'll press forward with 3..0.2.

**Comment by curtishall** [comment]
Resolved in the latest unstable release (3.0.4)
